testing/nitro/APKBUILD

```diff
@@ -0,0 +1,34 @@
+# Contributor: Bradley J Chambers <brad.chambers@gmail.com>
+# Maintainer: Bradley J Chambers <brad.chambers@gmail.com>
+pkgname=nitro
+pkgver=2.7
+_pkgver="$pkgver"dev-1
+pkgrel=0
+pkgdesc="The Nitro NITF project"
+url="https://github.com/hobu/nitro"
+arch="all"
+license="GPL3"
+depends="libstdc++ libgcc musl"
```

Comment:
Are those dependencies really needed? They should be automatically detected.

Reply:
They are automatically detected:
https://travis-ci.org/alpinelinux/aports/builds/252150974#L7304-L7306

Reply:
Thanks for catching this. I'll make the change.